phpstan.neon

```diff
@@ -0,0 +1,14 @@
+parameters:
+  scanDirectories:
+    - lib/
+    - lizmap/vendor/
+  level: 0
```

Comment:
Have you tried increasing the level ?
I saw it's from 0 to 7, right ?
What would be the advantage of increasing ?
Should we increasse step by step because there is too much error at level N+1 ?

Reply:
Yes, increasing the level brings a lot of error, doing it progressively seems a better idea.
Level 1 checks 'possibly' undefined variables so it will introduce a lot of false positive and level >= 2 checks methods and types based on PHPDocs so we will first need to do/complete rigourously all of them if we want it to be effective.